com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs

```diff
@@ -83,7 +83,8 @@ public enum SegmentationKind
                 if (frameStart == null || frameStart > frameCount) return;
 
                 timesSegmentationImageReceived++;
-                CollectionAssert.AreEqual(Enumerable.Repeat(expectedPixelValue, data.Length), data);
+                for (var i = 0; i < data.Length; i++)
```

Comment:
This check is not exactly the same as CollectionAssert.AreEqual, as it does not ensure the lengths of the collections are the same. It is also less explicit. I would suggest just calling .ToArray() on the NativeArray instead.

Reply:
Eyyy, this is why they pay ya the big bucks Jon. I'll implement your suggestion.